packages/extension/src/shared/wallet.model.ts

```diff
@@ -9,7 +9,14 @@ export interface WithSigner {
   signer: WalletAccountSigner
 }
 
-export interface WalletAccount extends WithSigner {
+export interface UniqAccount {
```

Comment:
it's not more unique than the other interface no? anyway avoid abbreviations

```suggestion
export interface BaseWalletAccount {
```